frontend/packages/ux-editor/src/components/config/expressions/DynamicContent.module.css

```diff
@@ -43,3 +43,24 @@
 .bold {
   font-weight: bold;
 }
+
+.complexExpression {
+  height: 150px;
```

Comment:
What about changing this to min-height: 150px? Would that be more responsive if the content of the complex expression grows bigger than 150px? 🤔

Reply:
Maybe! My initial though was that it was cleaner to adapt to scroller if the expression should grow, but potato, potato (ikke optimalt uttrykk å bruke tekstlig haha), I guess 😅